bundle/jsky.app.ot/src/main/scala/jsky/app/ot/gemini/editor/targetComponent/details/package.scala

```diff
@@ -26,6 +29,13 @@ package object details {
     override def textBoxAction(tbwe: TextBoxWidget): Unit = f(tbwe.getValue)
   }
 
+  def searchButton(doSearch: => Unit): JButton =
+    FlatButtonUtil.create("eclipse/search.gif") <| { b =>
+      b.addActionListener(new ActionListener() {
+        override def actionPerformed(e: ActionEvent) = doSearch
+      })
+    }
```

Comment:
I'm not sure whether this belongs here but wanted to share the implementation across the two usages.